gnupg.py

```diff
@@ -723,6 +723,35 @@ def handle_status(self, key, value):
         else:  # pragma: no cover
             logger.debug('message ignored: %s, %s', key, value)
 
+class AddSubKey(object):
+    "Handle status messages for --quick-add-key"
+
+    returncode = None
+
+    def __init__(self, gpg):
+        self.gpg = gpg
+        self.type = None
+        self.fingerprint = None
+        self.status = None
+
+    def __nonzero__(self):
+        if self.fingerprint: return True
```

Comment:
better written as `return bool(self.fingerprint)`

Reply:
I honestly copied that from the `GenKey` result. But I'm happy to change that

Reply:
Possibly the `GenKey` result needs changing, too ... there's some pretty old code in there!